staging/copy.sh

```diff
@@ -186,6 +187,7 @@ find "${CLIENT_REPO_TEMP}" -type f \( \
 
 echo "remove cyclical godep"
 rm -rf "${CLIENT_REPO_TEMP}/_vendor/k8s.io/client-go"
+rm -rf "${CLIENT_REPO_TEMP}/staging"
```

Comment:
The recursive staging folder contains the packages that are missing from my regular GOPATH. So during `godep save`, godep went into the newly added GOPATH and found the dependencies. When godep calculated what directory to copy these packages to, it used the regular GOPATH as the base, so these packages ended up at vendor/k8s.io/kubernetes/staging/...

So I think it's fine to just remove this recursive staging folder.

The detailed layout of the recursive staging folder:
```
~/go-workspace/.../k8s.io/kubernetes$ pwd
/Users/xuchao/go-workspace/src/k8s.io/kubernetes/staging/src/k8s.io/_tmp/vendor/k8s.io/kubernetes
~/go-workspace/.../k8s.io/kubernetes$ 
~/go-workspace/.../k8s.io/kubernetes$ tree staging/ -d
staging/
└── src
    └── k8s.io
        ├── apimachinery
        │   └── pkg
        │       └── util
        │           ├── mergepatch
        │           └── validationchao
        └── client-go
            ├── rest
            │   └── watch
            ├── third_party
            │   └── forked
            │       └── golang
            │           └── template
            └── util
                ├── cert
                ├── clock
                ├── flowcontrol
                ├── homedir
                ├── integer
                ├── jsonpath
                ├── testing
                └── workqueue

23 directories
```